src/modules/Progress/Progress.js

```diff
@@ -0,0 +1,97 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+import $ from 'jquery';
+import META from 'src/utils/Meta';
+
+export default class Progress extends Component {
+  static propTypes = {
+    autoSuccess: PropTypes.bool,
+    children: PropTypes.node,
+    className: PropTypes.string,
+    label: PropTypes.string,
```

Comment:
`label` only accepts two values, this should be `PropType.oneOf('percent', 'ratio')`.  Unless this prop is referring to a progress element label, and not the plugin property `label`.  In that case, there is a name conflict and we should fix it.